### ISSUE TITLE
style: 82 end-of-line comments explain 'what' instead of 'why'

### ISSUE THREAD

**Issue by CKickertz** [forkwright/aletheia#1639]
82 end-of-line comments explain what the code does rather than why. Per standards: "End-of-line comments explaining what a line does: rename the variable instead."

Mostly in test code (mneme test files, symbolon tests, nous session tests). Examples:

\`\`\`rust
tracker.track_sequence(&good_seq(), "s1", "nous1"); // same session
let volatility = 0.5; // neutral: no distortion
ctx.proximity.insert("alice".to_owned(), Some(0)); // seed
let mut child = guard.detach(); // disarms kill-on-drop
\`\`\`

Fix: either make the code self-documenting (rename variables, add named constants) or convert to \`// NOTE:\` if the context is genuinely non-obvious.
